components/Button/Button.js

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export function Button({ series }) {
+  const seriesID = series.id;
+  return (
+    <>
+      <Link href={`/series/${seriesID}`} passHref>
+        <a>
+          <StyledButton>Read more</StyledButton>
+        </a>
+      </Link>
+    </>
+  );
+}
+
+const StyledButton = styled.button`
```

Comment:
If it is wrapped in a link, it can look like a button, but it should not be an actual `<button>`.
Id suggest styling the `a` here. Maybe then a more meaningful component name would be `LinkButton`

```suggestion
const Anchor = styled.a`
```